pkg/local_object_storage/writecache/persist.go

```diff
@@ -64,6 +65,7 @@ func (c *cache) persistToCache(objs []objectInfo) []int {
 			}
 			sz += uint64(len(objs[i].data))
 			doneMem = append(doneMem, i)
+			storagelog.Write(c.log, objs[i].addr, "db PUT")
```

Comment:
Aren't those objects are also deleted from memory?

Reply:
Only if the method is called from `persistLoop`. I'll add log there.